core/controlplane/config/config.go

```diff
@@ -643,6 +644,8 @@ type Cluster struct {
 	HostedZoneID           string `yaml:"hostedZoneId,omitempty"`
 	ProvidedEncryptService EncryptService
 	CustomSettings         map[string]interface{} `yaml:"customSettings,omitempty"`
+	ExportKubeResources    bool                   `yaml:"exportKubeResources,omitempty"`
+	KubeResourcesS3Path    string                 `yaml:"-"`
```

Comment:
Shouldn't the annotation be `yaml:"kubeResourcesS3Path,omitempty"` or is there an expression like "-" which I'm not familiar with?